spec/features/image_upload_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'upload an image', type: :feature do
+  let!(:pass) { 'pass' }
+  let!(:user) { create(:user, password: pass, password_confirmation: pass) }
+
+  context 'when user logged in and add new card' do
+    
+    before(:each) do
+      visit login_path
+      fill_in :user_email, with: user.email
+      fill_in :user_password, with: pass
+      
+      click_button 'Вход'
+
+      visit root_path
+      click_link 'Добавить карточку'
+      
+      fill_in :card_original_text, with: 'new card'
+      fill_in :card_translated_text, with: 'новая карточка'
+
+      page.attach_file(:card_avatar, 'spec/support/help.png')
+
+      click_button 'Сохранить'
+    end
+
+    it "observes notice 'Новая карточка добавлена'" do
+      expect(page).to have_content 'Новая карточка добавлена'
+    end
+
+    it "card model attr contains avatar" do

```

Comment:
i think what tou can check what you
1.Haven't image at card
Add Image
2. Check what you have image at card

because if i broke functionality of adding card from UI, but add default image with name `help.png`, test will be passed, but functionality will not work correct.